### ISSUE TITLE
Not use round corners on block input and output.

### ISSUE THREAD

**Issue by rgaiacs** [carpentries/styles#95]
White space between code block and right bar.

![screenshot_2016-08-27_14-31-16](https://cloud.githubusercontent.com/assets/1506457/18027730/94e9c9aa-6c63-11e6-9d08-0e8d1307cb3f.png)

Suggestion 1:

![round](https://cloud.githubusercontent.com/assets/1506457/18027735/ad40df20-6c63-11e6-8b7d-6283c4e24f11.png)

This can be implemented with

```
div.python.highlighter-rouge pre.highlight {
    border-top-left-radius: 0;
    border-bottom-left-radius: 0;
}
```

Suggestion 2:

![square](https://cloud.githubusercontent.com/assets/1506457/18027737/b20cf61a-6c63-11e6-83b0-d33e65ad7092.png)

**Comment by colinmorris** [comment]
FWIW, I believe this can be closed, since it was fixed in #133

**Comment by maxim-belkin** [comment]
> FWIW, I believe this can be closed, since it was fixed in #133

Agreed. This issue has been fixed.
